pkgs/games/crawl/crawl_purify.patch

```diff
@@ -1,28 +1,21 @@
 diff --git a/crawl-ref/source/util/find_font b/crawl-ref/source/util/find_font
-index f8b576fd63..b95c21c0a1 100755
+index 9ba151e982..496e2cf2bf 100755
 --- a/crawl-ref/source/util/find_font
 +++ b/crawl-ref/source/util/find_font
-@@ -1,6 +1,6 @@
- #! /bin/sh
- 
+@@ -7,7 +7,7 @@
+ # some hard-coded font directories to check. Note: if you are using
+ # font-config with other directories, you don't need to edit this -- that case
+ # is handled by the `fc-list` call below.
 -FONTDIRS="/usr/share/fonts /usr/local/share/fonts /usr/*/lib/X11/fonts"
 +FONTDIRS="${fontsPath}/share/fonts"
```

Comment:
\Does it  work?

Reply:
While I haven't played extensively, all game text I've encountered renders correctly on my machine.

If you mean "Does the weird bash script this is patching do what it is intended to do", then I don't know.

Reply:
You should.

Usually we use `@var@` plus substituteAll.